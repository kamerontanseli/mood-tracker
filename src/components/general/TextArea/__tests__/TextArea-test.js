import React from 'react'
import renderer from 'react-test-renderer'
import Textarea from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <Textarea />
    );
    expect(snapshot).toMatchSnapshot()
  })
})