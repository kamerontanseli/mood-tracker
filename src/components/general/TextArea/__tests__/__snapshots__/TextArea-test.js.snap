// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<textarea
  className="TextArea"
/>
`;
